src/actions/article.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import dotenv from "dotenv";
+
+import {
+  CREATE_ARTICLE_SUCCESS,
+  CREATE_ARTICLE_FAILURE,
+  CREATE_ARTICLE_LOADING
+} from "../actionTypes/auth";
+
+dotenv.config();
+
+const setArticleSuccess = article => ({
+  type: CREATE_ARTICLE_SUCCESS,
+  payload: article
+});
+
+const setArticleLoading = isLoading => ({
+  type: CREATE_ARTICLE_LOADING,
+  payload: isLoading
+});
+
+const setArticleFailure = error => ({
+  type: CREATE_ARTICLE_FAILURE,
+  payload: error
+});
+
+export const createArticle = articleData => dispatch => {
+  const { token } = localStorage;
+
+  dispatch(setArticleLoading(true));
+  return axios
+    .post("https://thor-ah-staging.herokuapp.com/api/articles", articleData, {
```

Comment:
The Base API URL, "https://thor-ah-staging.herokuapp.com" should not be hard-coded. In order to allow you write your code your development environment you should extract this to a `process.env.REACT_APP_API` and add ```
REACT_APP_API='http://localhost:3000'
``` to your `.env`.

This way you can have dynamic base API URL's